backend/core/admin_database.py

```
@@ -273,6 +915,64 @@
+                        event_type,
+                        identifier,
+                        details,
+                        severity,
+                        ip_address,
+                        (user_agent[:500] if user_agent else None),
+                        datetime.now(),
+                    ),
+                )
+                return True
+        except Exception as e:
+            logger.error(f"Error recording security event: {str(e)}", exc_info=True)
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)